.github/workflows/main-build.yml

```diff
@@ -344,4 +345,4 @@ jobs:
           .platforms[.type == 'docker'].target_image_source := 'https://viash-hub.com/openpipelines-bio/openpipeline'
         platform: docker
         src: ${{ matrix.component.dir }}
-        setup: push
+        setup: push
```

Comment:
Could you undo this change?

Reply:
I would if I knew how - I restored the file but this minus sign keeps popping up in the diff, do you happen to know what it is?

> Could you undo this change?

I would if I knew how - I restored the file but this minus sign keeps popping up in the diff, do you happen to know what it is?